src/Aggregation/PercentilesBucket.php

```diff
@@ -69,12 +71,4 @@ public function setPercents(array $percents): self
     {
         return $this->setParam('percents', $percents);
     }
-
-    /**
-     * Set keyed flag to return the range as an hash instead of an array of key-value pairs.
-     */
-    public function setKeyed(bool $keyed): self
```

Comment:
Little BC break here, but not so important. So can probably be ignored. ping @ruflin @thePanz for confirmation?

Reply:
Maybe just adding an entry to the BC breaks section in the changelog is fine.

Reply:
How can it break BC?

Reply:
Previous method signature was different than the new one (facultative argument now in the trait).

This breaks when overriding method in a subclass.
See https://3v4l.org/FPiOB for exemple.

Reply:
Ah okay, you're right.

Reply:
fixed